blog/app/views/articles/index.html.haml

```diff
@@ -0,0 +1,16 @@
+%h1 Listing Articles
+= link_to 'New article', new_article_path
+%table
+  %tr
+    %th Title
+    %th Text
+    %th{:colspan => "3"}
+  - @articles.each do |article|
+    %tr
+      %td= article.title
+      %td= article.text
+      %td= link_to 'Show', article_path(article)
+      %td= link_to 'Edit', edit_article_path(article)
+      %td= link_to 'Destroy', article_path(article), |
```

Comment:
this probably just came from the html haml generator but you should put this all on one line and remove the pipes.